var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -23,6 +23,7 @@ class Dealii(CMakePackage, CudaPackage):
     transitive_rpaths = False
 
     version('develop', branch='master')
```

Comment:
```suggestion
    version('master', branch='master')
```

Reply:
Is this necessary? Will I now have to write `spack install dealii@master`, or will `spack install dealii@develop` still work?

Reply:
Maybe let me rephrase what I wrote above: Is this change being implemented throughout spack?

Reply:
Once upon a time, master was considered older than all stable releases. For this reason, a lot of packages were forced to rename master to develop for things to be handled correctly. This has since been fixed, and master is treated similar to develop, so the rename is no longer necessary. I’ve been slowly trying to update packages across Spack to call master 'master'. You would have to write `dealii@master`.